src/lnp/java/teammates/lnp/cases/BaseLNPTestCase.java

```diff
@@ -115,7 +115,7 @@ private String getPathToTestStatisticsResultsFile() {
                         this.getClass().getSimpleName(), this.timeStamp);
     }
 
-    private String createFileAndDirectory(String directory, String fileName) throws IOException {
+    String createFileAndDirectory(String directory, String fileName) throws IOException {
```

Comment:
Any reason for the default access modifier? Could we go with `protected` instead?

Reply:
You are right, I will change it